services/src/main/java/org/keycloak/authentication/authenticators/x509/CertificateValidator.java

```diff
@@ -468,6 +469,22 @@ public CertificateValidator validateExtendedKeyUsage() throws GeneralSecurityExc
         return this;
     }
 
+    public CertificateValidator validateTimestamps(boolean isValidationEnabled) throws GeneralSecurityException {
```

Comment:
Should we not validate the not-before and not-after on all certs in the chain rather than just the last one?

Reply:
not sure if this might cause other problems in some PKI's. That is why I just validated the last one.

Reply:
done